sys/log/include/log/log.h

```diff
@@ -51,8 +51,9 @@ struct log_handler {
 
 struct log_entry_hdr {
     int64_t ue_ts;
-    uint16_t ue_level;
     uint16_t ue_module;
+    uint16_t ue_index;
+    uint8_t ue_level;
```

Comment:
This is unaligning the structure, please align the structure.  you can add a ue_arg0, or unused field if you want to, but the behavior should be defined here.